com.unity.netcode.adapter.utp/.editorconfig

```diff
@@ -0,0 +1,106 @@
+root = true
+
+[*]
+charset = utf-8
+
+[*.cs]
+# spacing rules
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+# formatting rules
+dotnet_diagnostic.IDE0001.severity = error # Simplify name
+dotnet_diagnostic.IDE0002.severity = error # Simplify member access
+dotnet_diagnostic.IDE0005.severity = error # Remove unnecessary import
+dotnet_diagnostic.IDE0063.severity = error # Use simple 'using' statement
```

Comment:
We should be careful about this one. There's [a bug](https://fogbugz.unity3d.com/f/cases/1383971/) in IL2CPP related to its handling of the simpler version of the `using` statement. We got bit by that in UTP and decide to revert all of our uses of the simpler version to the older and more verbose version.

Reply:
I checked compiled IL and `using (var ..` generates idential IL to `using var ...`

Reply:
well :P if a C#8.0 syntactic sugar would manifest itself as a bug somehow in the NGO SDK, I'd like to have it.